modules/creating-a-machine-pool-cli.adoc

```diff
@@ -0,0 +1,106 @@
+----
+$ rosa create machinepool --cluster=<cluster-name> \
+                          --name=<machine_pool_id> \ <1>
+                          --enable-autoscaling \ <2>
+                          --min-replicas=<minimum_replica_count> --max-replicas=<maximum_replica_count> \ <3>
+                          --instance-type=<instance_type> \ <4>
+                          --labels=<key>=<value>,<key>=<value> <5>
+----
+<1> Specifies the name of the machine pool. Replace `<machine_pool_id>` with the name of your machine pool.
+<2> Enables autoscaling in the machine pool to meet the deployment needs.
+<3> Defines the minimum and maximum compute node limits. The cluster autoscaler does not reduce or increase the machine pool node count beyond the limits that you specify. If you deployed ROSA using a single availability zone, these options define the autoscaling limits in the machine pool for the zone. If you deployed your cluster using multiple availability zones, the options define the autoscaling limits in total across all zones and the counts must be multiples of 3.
+<4> Sets the instance type for the compute nodes in your machine pool. The instance type defines the vCPU and memory allocation for each compute node in the pool. Replace `<instance_type>` with an instance type. The default is `m5.xlarge`.
+<5> Defines the labels for the machine pool. Replace `<key>=<value>,<key>=<value>` with a comma-delimited list of key-value pairs, for example `--labels=key1=value1,key2=value2`.
++
+The following example creates a machine pool called `mymachinepool` that uses the `m5.xlarge` instance type and has autoscaling enabled. The minimum compute node limit is 3 and the maximum is 6 overall. The example also adds two workload-specific labels:
++
+[source,terminal]
+----
+$ rosa create machinepool --cluster=mycluster --name=mymachinepool --enable-autoscaling --min-replicas=3 --max-replicas=6 --instance-type=m5.xlarge --labels=app=db,tier=backend
+----
++
+.Example output
+[source,terminal]
+----
+I: Machine pool 'mymachinepool' created successfully on cluster 'mycluster'
+I: To view all machine pools, run 'rosa list machinepools -c mycluster'
+----
+
+.Verification
+
+. Verify that the machine pool is included in the list of available pools in your cluster:
++
+[source,terminal]
+----
+$ rosa list machinepools --cluster=<cluster_name>
+----
++
+.Example output
+[source,terminal]
+----
+ID             AUTOSCALING  REPLICAS  INSTANCE TYPE  LABELS                  TAINTS    AVAILABILITY ZONES                    SPOT INSTANCES
+Default        No           3         m5.xlarge                                        us-east-1a, us-east-1b, us-east-1c    N/A
+mymachinepool  Yes          3-6       m5.xlarge      app=db, tier=backend              us-east-1a, us-east-1b, us-east-1c    No
+----
+
+[NOTE]
+====
+For a detailed list of the arguments that are available for the `rosa create machinepool` subcommand, see _Managing objects with the rosa CLI_.
```

Comment:
The "Additional resources" list link that appears directly after this in the assembly is very similar to this note. Maybe nix it here? Especially since this one isn't able to be a real xref.

Reply:
Should `_Managing objects` here be a doc url? Per my understanding, it should be a doc

Reply:
> The "Additional resources" list link that appears directly after this in the assembly is very similar to this note. Maybe nix it here? Especially since this one isn't able to be a real xref.

Great suggestion, thank you. I have subsequently removed the note from this module and updated the additional resources reference.

> Should Managing objects with the rosa CLI be a url here in the NOTE? Per my understanding it should be a url

Currently, xrefs are not supported in modules, which is why it is occasionally necessary to reference a title that exists elsewhere in the assembly in this way. However, based on Alex's suggestion, the note is no longer present anyway.